articles/tutorials/04_working_with_textures/index.md

```diff
@@ -0,0 +1,364 @@
+---
+title: "Chapter 04: Working with Textures"
+description: In this chapter, we'll compare loading a texture from file versus pre-processing it using the content pipeline and how to render the textures as visuals in your game.
+---
+
+Textures are images that you use in your game to represent the visual graphics to the player. In order to use them, you need to either create the texture in code or load an existing one from a file and then draw the texture using the `SpriteBatch`.
+
+## Loading a Texture
+
+To load a texture in your game, you can either load it directly from its file (*.png*, *.jpg*, etc) or you can pre-process it using the *content pipeline* and load it using the `ContentManager`.
+
+The *content pipeline* can be used for more than just pre-processing images as textures; you can use it to also load audio, fonts, effects, and 3D models, all of which will be covered in upcoming chapters. It can also be customized to load various other types of file formats, however these extension are not covered in this tutorial.
+
+Let's go over both methods of loading a texture, and then we can discuss the pros and cons of each one. First, we need an image to load. Perform the following:
+
+1. Locate the *Content* directory inside your game project directory
+2. Create a new folder inside the *Content* directory called *images*.
+3. Right-click the following image of the MonoGame logo and save it in the *Content/images* directory you just created with the filename *logo.png*
+
+<figure><img src="./images/logo.png" alt="Figure 4-1: MonoGame Horizontal Logo"><figcaption><p><strong>Figure 4-1: MonoGame Horizontal Logo</strong></p></figcaption></figure>
+
+### Loading From File
+
+First we'll go over the process of loading the texture directly from the file at runtime. Open the *Game1.cs* file in your project and add the following instance member to the class where the `GraphicsDeviceManager` and `SpriteBatch` instance member variables are:

```

Comment:
First we'll go over the process of loading the texture directly from the file at runtime. Open the *Game1.cs* file in your project and add 

```cs
private Texture2D _logo;
```
right next to where the `GraphicsDeviceManager` and `SpriteBatch` instance member variables are declared.

`Texture2D` is the type used to store a reference to 2D image data in MonoGame.